### ISSUE TITLE
[Spec] Spec that disabling untrusted network disables top-level navigations (_unfencedTop and popups)

### ISSUE THREAD

**Issue by gtanzer** [WICG/fenced-frame#165]

**Comment by gtanzer** [comment]
@domfarolino At least the _unfencedTop part of this, you said there were parts of the spec that had been missed before and that you would fix.

**Comment by domfarolino** [comment]
Then that should probably be a separate issue (i.e., spec'ing `_unfencedTop` itself). That seems unrelated to the meat of this issue, although I agree this issue would be blocked on the new one that should be assigned to me.

**Comment by gtanzer** [comment]
Done and assigned you there rather than here

**Comment by domfarolino** [comment]
I know I mentioned it should be assigned to me, but just want to mention it is unlikely I'd be able to get to it for a while, so it might be worth assigning to someone more active on the project in case they can get to it before I would in the future.
